tests/integration/components/project-settings/general-settings/jira-project-test.js

```diff
@@ -273,6 +273,8 @@ module(
           this.jiraProjects[0].name,
           'Selected project "name" is equal to first item name on dropdown list'
         );
+
+        return new Response(200, {}, this.jiraProjects[0].toJSON());
```

Comment:
check if response code should be `200` ? otherwise for `post` code should be `201`